### PR TITLE
improvement: Revamp CSS and DIV structure of Comment module

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Revamp CSS and DIV structure of Comment module.
- Replace float property with flex.
- Removed Extra divs and manage it's CSS.
- Also removed `clear: both` property CSS as we using flex property so no need this.

### Types of changes
- Improvements

### How has this been tested?
- Test all type of comments design and check if it is breaking layout.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

**Comment by premanshup** [comment]
@sagarb90 Just wondering why there are no changes in the Static files? Can you please once verify if any changes in the SCSS files for comment required?

**Comment by sagarb90** [comment]
@premanshup As we have move all the static CSS to dynamic CSS  for the comment module so their is no change in SCSS file.

**Comment by premanshup** [comment]
> @premanshup As we have move all the static CSS to dynamic CSS for the comment module so their is no change in SCSS file.

@sagarb90 Ah right! The PR is looking good 👍🏼

File: comments.php [hunk 1]
```diff
@@ -30,7 +30,6 @@
 	<?php astra_comments_before(); ?>
 
 	<?php if ( have_comments() ) : ?>
-		<div class="comments-count-wrapper">
```
Comment (premanshup) [inline comments.php:33]: @sagarb90 Need to manage backward compatibility for HTML markup with old structure.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: comments.php (hunk 1 above)
Comment (sagarb90) [reply comments.php:33]: Added  backward compatibility using common markup function.

**Review by premanshup — CHANGES_REQUESTED** [review]
@sagarb90 Instead of overriding the CSS, can we have just one CSS selector? Maybe we can add layout specific CSS Classes and add them conditionally.

For eg. - [https://d.pr/i/FdDAWj](https://d.pr/i/FdDAWj)

**Review by sagarb90 — COMMENTED** [review]
`Instead of overriding the CSS, can we have just one CSS selector? Maybe we can add layout specific CSS Classes and add them conditionally` - This is Done, Added layout condition for border-bottom CSS.

**Review by premanshup — APPROVED** [review]

**Pull request merged**